src/components/single-place/report-section/report-section.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import ReportModal from './report-modal/report-modal';
+import './report-section.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFlag } from '@fortawesome/free-regular-svg-icons';
+
+export default class ReportSection extends Component {
+  constructor(props) {
+    super(props)
+    this.hideModal = this.hideModal.bind(this)
```

Comment:
I think bind is bad idea) We have Es6)

Reply:
Yes, you can just delete it )You use arrow fuunction)

Reply:
Thank you for advise. Done.